paper/paper.md

```diff
@@ -0,0 +1,69 @@
+
+While some research code for these models exists, they are not focused on usability and limited to FORTRAN **TODO LINK** and proprietary software like MATLAB **TODO LINK**. There is a lack of well-documented, maintained open-source implementations, particularly in modern programming languages and with a focus on scalability and performance.
+
+The presented software has already seen application in several research works. It has been used to optimize trajectories around the highly irregular comet 67P/Churyumov-Gerasimenko [@marak2023trajectory]. Further, it has been used to study the effectiveness of so-called neural density fields [@izzo2022geodesy], where it can serve as a ground truth and to pretrain neural networks [@GeodesyNetsBenchmark]. **TODO_add_more_examples**
+
+Thus, overall this model is highly versatile and can be used in a wide range of applications. We hope it will enable further research in the field, especially related to recent machine learning techniques, which typically rely on Python implementations.
+
+# Polyhedral Model
+
+On a mathematical level, the implemented model follows the approach by Petrović [@petrovic1996determination] as refined by Tsoulis and Petrović [@tsoulis2001singularities]. A comprehensive description of the mathematical foundations of the model is given in the associated student report [@jonas_report]. 
+
+Implementation-wise it makes use of the inherent parallelization opportunity of the approach as it iterates over the mesh. This parallelization is achieved via *thrust* which allows utilizing *OpenMP* and *Intel TBB*. On a finer scale, individual costly operations were investigated and, e.g., the \texttt{arctan} operations were vectorized using *xsimd*. On an application side, the user may use the implemented caching mechanism to avoid recomputation of mesh properties, such as the face normals.
+
+Extensive tests using GoogleTest are used via GitHub Actions to ensure the (continued) correctness of the implementation.
+
+# Installation \& Contribution
+
+The library is available on GitHub \footnote{\url{https://github.com/esa/polyhedral-gravity-model}} and can be installed with *pip* \footnote{\url{https://github.com/esa/polyhedral-gravity-model#pip}} or from *conda* \footnote{\url{https://anaconda.org/conda-forge/polyhedral-gravity-model}}. Build instructions using *CMake* are provided in the repository. The library is licensed under a GPL license.
```

Comment:
> can be installed with *pip* \footnote{\url{https://github.com/esa/polyhedral-gravity-model#pip}}

@schuhmaj Did we ever consider properly putting it on pip? Would be nicest if people can just `pip install polyhedral-gravity-model`